arviz/stats/density_utils.py

```diff
@@ -1036,3 +1036,40 @@ def histogram(data, bins, range_hist=None):
     hist, bin_edges = np.histogram(data, bins=bins, range=range_hist)
     hist_dens = hist / (hist.sum() * np.diff(bin_edges))
     return hist, hist_dens, bin_edges
+
+
+def _find_hpd_contours(density, hpd_levels):
+    """
+    Find contours enclosing regions of highest posterior density.
+
+    Parameters
+    ----------
+    density : array-like
+        A gridded 2D KDE.
```

Comment:
I would be explicit in that we need a grid where each value has the same area underneath. The area doesn't need to be a square, but it needs to be the same for all values.

Reply:
```suggestion
        A 2D KDE on a grid with cells of equal area.
```
Would this suffice?

Reply:
yes